forms/edit_profile.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+from wtforms_alchemy import ModelForm
+
+from wtforms.validators import Regexp
+
+from flask_wtf import FlaskForm
+
+from models import User
+
+
+email_regex = Regexp('[0-9A-Za-z-.]*@tieto.com$',
```

Comment:
if such validator/regex exist anywhere else in the project, try to import it

Reply:
I've tried to do this way at the beginning, but had some obstacles. I am afraid that my ModelForm is not compatible with others'.